src/Resources/contao/templates/backend/be_no_root.html5

```diff
@@ -16,8 +16,14 @@ $lang = (object) $GLOBALS['TL_LANG']['XPT'];
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
 
   <style><?php
-    include TL_ROOT . '/system/themes/' . $theme . '/basic.css';
-    include TL_ROOT . '/system/themes/' . $theme . '/error.css';
+    $includeIfExists = function ($file) {
```

Comment:
So what? We are only talking about error templates.

It was also possible to disable modules in the backend. And it was possible to define the URL suffix and some other settings. We did not care about this either, did we?